common/librato/librato.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
I was planning to, but https://www.librato.com/docs/api/#annotations isn't ready for tags yet. I'll contact librato to find out a timeframe.